src/styles.css

```diff
@@ -0,0 +1,100 @@
+body {
+	background-color: #161516;
+	color: white;
+}
+
+/* NAVBAR */
+nav {
+	background-color: #333;
+	padding-right: 50px;
+	font-family: Helvetica;
+}
+
+#navTitle {
+	float: left;
+	margin: 10px 0 0 150px;
+	color: white;
+	font-size: 28px;
+	font-family: Georgia, serif;
+}
+
+#navList {
+	list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+.nav-element {
+	float: right;
+	margin-right: 20px;
+	font-size: 26px;
+}
+
+.nav-element {
+	display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
```

Comment:
You seem to have the duplicate class here.
Also, I would suggest using the `cursor: pointer;` here too.

![navcursorres](https://user-images.githubusercontent.com/36057474/72870770-62bfb980-3ce9-11ea-88d2-a995b099525c.gif)